make-debs.sh

```diff
@@ -19,7 +19,7 @@ set -xe
 base=${1:-/tmp/release}
 codename=$(lsb_release -sc)
 releasedir=$base/$(lsb_release -si)/WORKDIR
-rm -fr $releasedir
+rm -fr $base/(lsb_release -si)
```

Comment:
could you elaborate a little bit what "2nd time build error" stands for?

Reply:
@tchaikov The error is below:
_File "pool/main/c/ceph/rbd-nbd_14.0.1-927-g004d918ae0-1_amd64.deb" is already registered with different checksums!
md5 expected: bb874edadc09b586d8947780ccf32757, got: cfc00c331cdb81ffe9732883355a99dd
sha1 expected: 59f6e5c1905b05369f0d7780fc1ed5aa1d9ea0f4, got: 36bf0b427dd4b29625cb5b19e874cfa61688cb02
sha256 expected: 2e0f7486e0ae379fe16b3efc414a0677e264b13af12070dd4712b32eca94b042, got: 9bd22cfb676572afa643f20ba1a2d50489a312faf5065322ae3638db9e2cded8
size expected: 121452, got: 121496
There have been errors!_

Reproduce steps:
1. do_cmake.sh
2. cd build; make; cd -
3. make-debs.sh #no error
4. change source code and build again "cd build; make; cd -"
5. make-debs.sh #this above error